.yamllint.yml

```diff
@@ -0,0 +1,2 @@
+rules:
+  document-start: disable
```

Comment:
The document start marker (`---`) is not necessary in dbt files; the linter only emits this as a warning by default, but I found it annoying so I'm disabling it.